tests/unit/checkDeployBlockersTest.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @jest-environment node
+ */
+const core = require('@actions/core');
+const github = require('@actions/github');
+const run = require('../../.github/actions/checkDeployBlockers/checkDeployBlockers');
+
+// Static mock function for core.getInput
+const mockGetInput = jest.fn().mockImplementation((arg) => {
+    if (arg === 'GITHUB_TOKEN') {
+        return 'fake_token';
+    }
+
+    if (arg === 'ISSUE_NUMBER') {
+        return 1;
+    }
+});
+
+const mockSetOutput = jest.fn();
+const mockGetIssue = jest.fn();
+const mockListComments = jest.fn();
+
+beforeAll(() => {
+    // Mock core module
+    core.getInput = mockGetInput;
+    core.setOutput = mockSetOutput;
+
+    // Mock octokit module
+    const mocktokit = {
+        rest: {
```

Comment:
Not sure we need the `rest`. Since other actions are working without it, I'd suggest we nix it!